src/fusion_power_plant.cc

```diff
@@ -43,7 +44,12 @@ void FusionPowerPlant::EnterNotify() {
   fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / 
     (kDefaultTimeStepDur * 12) * context()->dt());
   blanket_turnover = blanket_size * blanket_turnover_fraction;
-  double startup_inventory = reserve_inventory + sequestered_equilibrium; 
+  startup_inventory = reserve_inventory + sequestered_equilibrium; 
+
+  // Otherwise tritium-limited sims will never startup. We need to overbuy at
+  // startup.
+  trituim_overbuy_multiplier = 1 + (tritium_yearly_decay_fraction / 
```

Comment:
1. you have a typo in `tritium`
2. this is only used in this function, does it need to be a member variable?
3. I think the name of this variable makes is hard to understand/remember why it exists. I wonder if we should instead defined `tritium_timestep_decay_fraction` and multply by `(1 + tritium_timestep_decay_fraction)` where necessary.

Reply:
I think I'm going to try to remove this whole thing with the over-purchasing of tritium, honestly. It's kind of confusing, and makes the code do things that aren't obvious or user-defined.